src/Webpay/Oneclick/MallTransactionCaptureResponse.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+
+namespace Transbank\Webpay\Oneclick;
+
+class MallTransactionCaptureResponse
+{
+    public $authorizationCode;
+    public $authorizationDate;
+    public $capturedAmount;
+    public $responseCode;
+
+    public function __construct($json)
+    {
+        $this->setAuthorizationCode($json["authorization_code"] ?? null);
+        $this->setAuthorizationDate($json["authorization_date"] ?? null);
+        $this->setCapturedAmount($json["captured_amount"] ?? null);
+        $this->setResponseCode($json["captured_amount"] ?? null);
```

Comment:
```suggestion
        $this->setAuthorizationCode(isset($json["authorization_code"] ? $json["authorization_code"] : null);
        $this->setAuthorizationDate(isset($json["authorization_date"]) ? $json["authorization_date"] : null);
        $this->setCapturedAmount(isset($json["captured_amount"]) ? $json["captured_amount"]  : null);
        $this->setResponseCode(isset($json["response_code"]) ? $json["response_code"] : null);
```

The setResponseCode says 'captured amount' and this operator (??) is only availablo on PHP 7.X :(